server/athenian/api/openapi/openapi.yaml

```diff
@@ -1371,7 +1371,7 @@ components:
       type: array
     Contributor:
       additionalProperties: false
-      description: Contributor details.
+      description: Details about a developer who contributed to some repositories owned by the account.
```

Comment:
Can't be added new hires to their team till they make a contribution? Why not listing also organization members?

Reply:
It will behave the same way as `/filter/contributors` just without the time constraint. New hires and org members won't be shown in the filtering as well as now. Is this limitation ok @warenlg ?

Reply:
Yes, it is ok, I don't want us to spend eng effort on new hires that have not contributed anywhere yet. The use case is too minor, and there is no need to handle it from our users.